Tasks/Kubernetes/task.json

```diff
@@ -0,0 +1,207 @@
+{
+    "id": "CBC316A2-586F-4DEF-BE79-488A1F503564",
+    "name": "Kubernetes",
+    "friendlyName": "Deploy to Kubernetes",
+    "description": "Deploy of run a command on kubernetes cluster",
```

Comment:
we dont install we download kubectl client. I will ask Atul to add this in the Notes